contracts/interchain-token-service/src/testutils.rs

```diff
@@ -1,2 +1,160 @@
-#![cfg(any(test, feature = "testutils"))]
+use soroban_sdk::testutils::Address as _;
+use soroban_sdk::token::StellarAssetClient;
+use soroban_sdk::{Address, BytesN, Env, IntoVal, String, Vec};
+use soroban_token_sdk::metadata::TokenMetadata;
+use stellar_axelar_gas_service::{AxelarGasService, AxelarGasServiceClient};
+use stellar_axelar_gateway::testutils::{
+    generate_proof, get_approve_hash, setup_gateway, TestSignerSet,
+};
+use stellar_axelar_gateway::types::Message;
+use stellar_axelar_gateway::AxelarGatewayClient;
+use stellar_axelar_std::types::Token;
+
+use crate::{InterchainTokenService, InterchainTokenServiceClient};
+
+#[cfg(any(test, feature = "testutils"))]
```

Comment:
this feature gating isn't needed since it's already done in `lib.rs` for the entire module